_includes/head.html

```diff
@@ -3,9 +3,9 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {%- seo title=false -%}
-  <link rel="stylesheet" href="{{ site.url }}/assets/css/main.css" />
+  <link rel="stylesheet" href="/assets/css/main.css" />
   <link rel="shortcut icon" type="image/x-icon" href="{{ site.favicon }}" />
-  <link rel="stylesheet" href="{{ site.url }}/assets/css/agate.css" />
+  <link rel="stylesheet" href="/assets/css/agate.css" />
```

Comment:
Please keep the related path of your site, or it cannot find the .css files when deployed.